TROUBLESHOOT.md

```diff
@@ -0,0 +1,165 @@
+# Debug Manual Flink - BigQuery Connector
+
+## Overview
+When problems occur, debug tools enable users to quickly pinpoint the root cause, whether it's 
+within the connector itself, the BigQuery API, or the network infrastructure.
+Detailed logs, traces, and debugging capabilities help you effectively troubleshoot issues, 
+reducing the time and effort required to resolve them.
+This document is a debugging manual highlighting possible issues that could be faced by the users 
+and how to troubleshoot them. It also aids in identifying the error causes and proposed steps for 
+mitigating them.
+
+<i>Note: This is a running document containing the issues reported to the Developers of the 
+[Flink - BigQuery Connector](https://github.com/GoogleCloudDataproc/flink-bigquery-connector). 
+Please feel free to contribute as per any new issues that may arise.</i>
+
+## Flink Metrics
+### Writing to BigQuery
+As an effort to increase observability, the Flink-BigQuery Connector Team provides support to 
+collect and report Flink Metrics for a Flink Application.
+The details of the metrics supported so far are available in the 
+[README](https://github.com/GoogleCloudDataproc/flink-bigquery-connector/blob/main/README.md#flink-metrics).
+
+An overview of records seen by the writer, number of records successfully written to 
```

Comment:
An overview of records seen by the writer `and` number of records successfully ....

Reply:
Done.